sse2neon.h

```diff
@@ -9533,6 +9533,17 @@ FORCE_INLINE __m128i _mm_aesenc_si128(__m128i EncBlock, __m128i RoundKey)
 #endif
 }
 
+FORCE_INLINE __m128i _mm_aesdec_si128(__m128i a, __m128i RoundKey)
+{
+#if defined(__aarch64__)
```

Comment:
You shall take the following into consideration:
1. Armv8-A without Cryptographic extension
2. [Armv8-A and A32 with Cryptographic extension](https://developer.arm.com/documentation/100242/0100/functional-description/about-the-cryptographic-extension)
3. Armv7-A

Reply:
It seems we don't have tests for function guarded by "__ARM_FEATURE_CRYPTO" macro. I think maybe I should add another makefile target to run test with `+crypto`?

Reply:
Exactly, we shall have conditional `+crypto` in feasible permutations.